Gems/PhysX/Code/Tests/CharacterGameplayControllerTests.cpp

```diff
@@ -124,4 +133,33 @@ namespace PhysX
             EXPECT_THAT(expectedGravityMultiplier + i, testing::FloatEq(basis.m_gameplayController->GetGravityMultiplier()));
         }
     }
+
+    TEST_F(PhysXDefaultWorldTest, CharacterGameplayController_EntityFallsUnderGravity)
+    {
+        GameplayTestBasis basis(m_testSceneHandle, DefaultGravityMultiplier, DefaultGroundDetectionBoxHeight, DefaultFloorTransform);
+        auto* sceneInterface = AZ::Interface<AzPhysics::SceneInterface>::Get();
+        sceneInterface->RemoveSimulatedBody(basis.m_sceneHandle, basis.m_floor->m_bodyHandle);
```

Comment:
A comment indicating why the floor rigid body is being deleted would help to understand more the test.

Reply:
Refactored to test basis and added a docstring to the function as well as a comment to the test.